### PR TITLE
Feature/gramine

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

**Review by mcornaton — APPROVED** [review]
Thanks! :)

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 version=7.1.1
-iexecCommonVersion=6.0.0
+iexecCommonVersion=6.0.0-NEXT-SNAPSHOT
```
Comment (jeremyjams) [inline gradle.properties:2]: Why do we need next?

File: gramine/entrypoint.manifest [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+sgx.allowed_files.conf = "file:/workplace/conf/config.json"
+sgx.protected_files.iexecout = "file:/iexec_out"
+sgx.protected_files.post-compute-tmp = "file:/post-compute-tmp"
+
+loader.log_level =  "none"
+#************for premain*********************
+sgx.allowed_files.manifest = "file:/entrypoint.manifest.sgx"
+libos.entrypoint = "file:/workplace/app/premain"
+#JAVA: bin path
+app.main="/usr/bin/java"
```
Comment (jeremyjams) [inline gramine/entrypoint.manifest:63]:
```suggestion
app.main="/usr/bin/java"

```

**Review by jeremyjams — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (jbern0rd) [reply gradle.properties]: I don't know, it was already done when I inherited from the branch.
I reverted this change in 70ff3c3.

File: gramine/entrypoint.manifest (hunk 2 above)
Comment (jbern0rd) [reply gramine/entrypoint.manifest]: Done in 8301d1e

**Pull request merged**